packages/frontend/src/components/accounts/ledger/LedgerConnectModal/views/Connect.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
```

Comment:
Do these need to be separate components? The only things different between the three is the icon and the translation strings, seems like they could be consolidated for better reuse.

If we think these will dramatically diverge in the future then it makes sense to keep them separate but if we don't anticipate many changes then it probably makes more sense to combine them into a single component that takes an `isError` prop and translation string values. What do you think?

Reply:
I'm suggesting passing in the text as separate props - `isError` would just be to determine the icon. Though the icon could be a separate prop as well.

Reply:
Implemented perfect solution. One component, no text in props, simpler wrapper:
https://github.com/near/near-wallet/pull/2595/commits/e7c209e9955ac960ad9386907d85987731e358e1